build.sbt

```diff
@@ -18,6 +18,7 @@ val druidVersion = "0.8.1"
 val finagleVersion = "6.25.0"
 val twitterUtilVersion = "6.25.0"
 val samzaVersion = "0.8.0"
+val sparkVersion = "1.4.1"
```

Comment:
I think Spark is on 1.5 now, could we use that or is there an issue preventing it?